src/components/common/Expenditures/ExpenditureForm/helpers.ts

```diff
@@ -12,6 +12,7 @@ export const getInitialPayoutFieldValue = (
   recipientAddress: '',
   tokenAddress,
   amount: '0',
+  claimDelay: 0,
```

Comment:
Nitpick. Should these two values e of the same type? Ie: either `int`'s or `string`'s

Reply:
There's an issue to store all `uint256` values into strings: https://github.com/JoinColony/colonyCDapp/issues/918